gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -694,6 +770,37 @@ gst_tensor_trainer_query_caps (GstTensorTrainer * trainer,
   return caps;
 }
 
+/**
+ * @brief Check if training is complete in subplugin to avoid deadlock.
+ * deadlock: is_training_complete is false and training ends in the subplugin after g_cond_wait().
+ */
+static gboolean
+gst_tensor_trainer_training_is_complete (GstTensorTrainer * trainer)
```

Comment:
This is also "wait" function, not "is_***" function.
"is_***" do not wait for the event. It simply returns with True or False without waiting.

Reply:
okay. thank you. I will change this.